src/components/Content.jsx

```diff
@@ -0,0 +1,59 @@
+import { useState } from 'react';
```

Comment:
If `Content` component is supposed to handle any kind of logic, I think it might be worth to think about some separate name and place for this type of a component. What do you think about `src/containers/Content.jsx`?

https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0